packages/terra-site/src/examples/date-picker/DatePickerDefault.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-default"
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
```

Comment:
The change would just be changing this in handleChange. I'll bring this up with the group to see if a change like this would be non-passive.
```javascript
date.format()
```
to
```javascript
date.format('YYYY-MM-DD')
```